scss/_reboot.scss

```diff
@@ -360,6 +360,18 @@ html [type="button"], // 1
   -webkit-appearance: button; // 2
 }
 
+// Opinionated: add "hand" cursor to non-disabled button elements.
+@if $enable-hand-cursor-for-buttons {
```

Comment:
as per @andresgalante's suggestion, maybe rename to `enable-pointer-cursor-for-buttons` - the preceding comment should make it clear what it means in practice